content/swagger/resources/reference_entity_records/definitions/reference_entity_record.yaml

```diff
@@ -3,11 +3,11 @@ required: ["code"]
 properties:
   code:
     type: string
-    description: Code of the record of a reference entity
+    description: Code of the record
```

Comment:
but it's not unique, which is different from the PIM. Uniqueness is "reference entity code / record code"
I don't know how to explain that.

Reply:
I'm not talking about the uniqueness of the attribute here. I just removed `of a reference entity` because, in the final display, the words `reference entity` appear way too many times.